src/tools/miri/src/diagnostics.rs

```diff
@@ -395,6 +395,7 @@ pub fn report_msg<'tcx>(
 ) {
     let span = stacktrace.first().map_or(DUMMY_SP, |fi| fi.span);
     let sess = machine.tcx.sess;
+    let title = title.to_string();
```

Comment:
This is now calling `to_string` on a `String`... am I missing something or is that just unnecessary?

Reply:
Good catch, I have removed it.